src/components/widgets/LunchWidget.tsx

```diff
@@ -0,0 +1,47 @@
+import moment from 'moment';
+import React, { useCallback, useContext } from 'react';
+import { useQuery } from 'react-apollo';
+import { NavigationScreenProp } from 'react-navigation';
+
+import { colors, texts } from '../../config';
+import { graphqlFetchPolicy } from '../../helpers';
+import { useHomeRefresh } from '../../hooks/HomeRefresh';
+import { lunch } from '../../icons';
+import { NetworkContext } from '../../NetworkProvider';
+import { getQuery, QUERY_TYPES } from '../../queries';
+import { DefaultWidget } from './DefaultWidget';
+
+type Props = {
+  navigation: NavigationScreenProp<never>;
+};
+
+export const LunchWidget = ({ navigation }: Props) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp });
```

Comment:
i miss a refreshTime here. now when i change the date back and forth it result in server query every time. this could be avoided with a refresh time for lunches to have lunch data cached.

Reply:
done in 3795523c9fe7f462df6678bc1c684c0c58969914